src/console/MeasurementCollectionToFhir/ProcessorStartup.cs

```diff
@@ -49,19 +49,32 @@ public void ConfigureServices(IServiceCollection services)
 
             services.TryAddSingleton<ResourceManagementService>();
             services.TryAddSingleton<IExceptionTelemetryProcessor, FhirExceptionTelemetryProcessor>();
+            
             services.TryAddSingleton<MeasurementFhirImportOptions>();
-            services.TryAddSingleton<MeasurementFhirImportService>();
+
+            var collectorOptions = new CollectorOptions();
```

Comment:
Suggestion: It would be better if this is a factory or a factory extension method that is part of the Events library.  This should allow easier reuse in PaaS.

Reply:
updated to use an extension here and in the other location where we reference the options and inject the appropriate class